qiskit/chemistry/results/eigenstate_result.py

```diff
@@ -32,12 +31,12 @@ def eigenvalue(self, value: complex) -> None:
         self.data['eigenvalue'] = value
 
     @property
-    def aux_values(self) -> Optional[np.ndarray]:
+    def aux_values(self) -> Optional[Dict[str, float]]:
```

Comment:
This needs to be reverted, too.

Reply:
thanks! is this actually an np.array or a list of floats? 🤔

Reply:
I honestly don't know why this would have been an `np.array` to begin with. I would go for a `List[float]`..

Reply:
changed to list[float] 👍